modules/client-configuration/pages/snippets/addchannels_vendor_cli.adoc

```diff
@@ -0,0 +1,16 @@
+.Procedure: Adding Software Channels at the Command Prompt
+. At the command prompt on the {productname} Server, as root, use the [command]``mgr-sync`` command to add the appropriate channels:
++
+----
+mgr-sync add \
+<base_channel_name> \
+<child_channel_name_1> \
+<child_channel_name_2> \
+... <child_channel_name_n>
+----
+. Synchronize the channels:
++
+----
+mgr-sync sync
```

Comment:
When you "add" a channel with `mgr-sync add channel` the sync is started automatically.
This sync command is only available if your later want to trigger a sync when the channels are already available.

Additionally the command requires more options.

`$> mgr-sync sync channel `

or 

`$> mgr-sync sync --with-children channel `

Reply:
I wonder: isn't this the same as modules/ROOT/pages/snippets/add_channels_mgrsync.adoc  ?

Reply:
Yes, I renamed it.